src/@next/Checkbox/Checkbox.tsx

```diff
@@ -40,6 +42,8 @@ export const Checkbox = React.forwardRef<HTMLDivElement, CheckboxProps>(
       hasError,
       helpText,
       isPadded = true,
+      isHovered = false,
```

Comment:
Why do we need this? 🤔

Reply:
it's to trigger hover when Checkbox table cell is hovered. Basically, it widens the area of point to trigger Checkbox hover. it's one of the requirements for the hover state

Reply:
we can't trigger checkbox hover with just html when the table cell is hovered though?